app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError(f"{visitor['name']} is not vaccinated!")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError(f"{visitor['name']}'s"
+                                       f" vaccine is expired!")
```

Comment:
I'm not sure if it's fine to write the name because you don't know if there just one friend hat doesn't wear a mask